1-JavaScript/Week3/homework/ex7-mindPrivacy.js

```diff
@@ -28,8 +28,14 @@ const employeeRecords = [
 ];
 
 // ! Function under test
-function filterPrivateData(/* parameter(s) go here */) {
-  // TODO complete this function
+function filterPrivateData(filteredRecords) {
+  const array=[];
+for (const obj of filteredRecords){
+   const {name,occupation,email}=obj;
+    const newObj={name,occupation,email};
+  array.push(newObj);
```

Comment:
You could simplify this a bit by doing this:
```suggestion
   const {name, occupation, email} = employeeRecord;
   array.push({name,occupation,email});
```

And you can even do the reversed thing, that is even more advanced: 
```
   const {gender, salary, ...publicData} = employeeRecord;
   array.push(publicData);
```